packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -236,6 +243,93 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem *)item
   return self;
 }
 
+- (void)setPictureInPicture:(BOOL)pictureInPicture {
+    if (self._pictureInPicture == pictureInPicture) {
+        return;
+    }
+
+    self._pictureInPicture = pictureInPicture;
+    if (@available(iOS 9.0, *)) {
+        if (_pipController && self._pictureInPicture && ![_pipController isPictureInPictureActive]) {
+            self._playerLayer.opacity = 0.001;
+            if (_eventSink != nil) {
+              _eventSink(@{@"event" : @"startingPiP"});
+            }
+            dispatch_async(dispatch_get_main_queue(), ^{
+                [_pipController startPictureInPicture];
+            });
+        } else if (_pipController && !self._pictureInPicture && [_pipController isPictureInPictureActive]) {
+            dispatch_async(dispatch_get_main_queue(), ^{
+                [_pipController stopPictureInPicture];
+            });
+        } else {
+            // Fallback on earlier versions
```

Comment:
what is earlier version

Reply:
This will be removed because current min version is already 9